src/neuroconv/__init__.py

```diff
@@ -1,4 +1,11 @@
 from .nwbconverter import NWBConverter
-from .datainterfaces import *
 from .tools import spikeinterface, roiextractors, neo
 from .tools.yaml_conversion_specification import run_conversion_from_yaml
+
+
+def __getattr__(name: str):
+    from importlib import import_module
+
+    interface_location = dict(DeepLabCutInterface="datainterfaces.behavior.deeplabcut")
```

Comment:
I'd prefer if this were outside the scope of the function as well

Reply:
That would be a good idea for portability, yes.